tooling/ssa_executor/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "noir_ssa_executor"
```

Comment:
It would be nice to merge #8135 first, to make this PR smaller. I tried to target that branch as base, but there are too many unrelated changes in this.